src/DDTrace/Integrations/Redis/RedisIntegration.php

```diff
@@ -0,0 +1,835 @@
+<?php
+
+namespace DDTrace\Integrations\Redis;
+
+use DDTrace\Integrations\Integration;
+use DDTrace\Tag;
+use DDTrace\Type;
+use DDTrace\GlobalTracer;
+use DDTrace\Util\TryCatchFinally;
+
+const VALUE_PLACEHOLDER = "?";
```

Comment:
I don't see where these constants are being used anywhere. I think they can be removed. :)